aggregator/src/aggregator/aggregation_job_driver.rs

```diff
@@ -899,29 +900,80 @@ impl AggregationJobDriver {
         self: Arc<Self>,
         datastore: Arc<Datastore<C>>,
         maximum_attempts_before_failure: usize,
-    ) -> impl Fn(Lease<AcquiredAggregationJob>) -> BoxFuture<'static, Result<(), anyhow::Error>>
-    {
+    ) -> impl Fn(Lease<AcquiredAggregationJob>) -> BoxFuture<'static, Result<(), Error>> {
```

Comment:
This function is basically the same as its collector counterpart, which makes it a candidate for hoisting out into the `JobDriver`, however I think it's just under the threshold where that refactor would be worth it, in terms of how much code needs to be touched.